src/components/sprite-selector-item/sprite-selector-item.jsx

```diff
@@ -39,6 +39,15 @@ const SpriteSelectorItem = props => (
         ) : null}
         <div className={styles.spriteName}>{props.name}</div>
         <ContextMenu id={`${props.name}-${contextMenuId++}`}>
+            {props.onDuplicateButtonClick ? (
+                <MenuItem onClick={props.onDuplicateButtonClick}>
+                    <FormattedMessage
+                        defaultMessage="duplicate"
+                        description="Menu item to duplicate in the right click menu"
+                        id="contextMenu.duplicate"
+                    />
+                </MenuItem>
+            ) : null}
```

Comment:
If neither `onDeleteButtonClick` nor `onDuplicateButtonClick` are required, then I think for consistency, it'd be good to have the prop check for both of these menu items.